src/Microsoft.AspNet.Mvc.TagHelpers/ModeMatchResult.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
```

Comment:
I put it here because it's the return type of the extension method in that namespace. No biggie either way.